docker/docker-compose.etna-app.shared.yml

```diff
@@ -29,6 +29,7 @@ x-base-etna-service:
   environment: &base-etna-env
     '${APP_ENV_PREFIX}ENV': 'development'
     APP_NAME: '${APP_NAME}'
+    METRICS_PW: 'metrics_pw'
```

Comment:
Any etna app that you want to expose metrics from will need this environment variable.  In production we'll use a long secret.  Eventually this goes away when our apps move over to the docker overlay network.